### ISSUE TITLE
The fine-tuned model does not converge.

### ISSUE THREAD

**Issue by CUITCHENSIYU** [facebookresearch/dinov2#163]
I am using the pretrained model provided here as the backbone for training my own dataset, which consists of sequential data. The difference is that my dataset is composed of temporal data. During training, I set the learning rate to 1e-4 and used the Adam optimizer. However, the model exhibits a lack of convergence from the first epoch.
